docs/developer/taxes.mdx

```diff
@@ -303,6 +303,20 @@ In this example, we are fetching a product list with prices for Poland.
 }
 ```
 
+### Rounding
+
+The following rounding rule is applied to the tax calculation:
+
+`If the discarded fraction is >= 0.5, round up; if the discarded fraction is < 0.5, then round down`
+
+Examples:
+
+`$9.945 => $9.95`
+
+`$9.94499 => $9.94`
+
+Precision for most currencies is calculated up to 2 decimal places.
```

Comment:
This is true for flat rates. With tax apps this can be different. Should it be mentioned here?